ingestion/src/metadata/ingestion/source/database/dbt_source.py

```diff
@@ -76,9 +76,10 @@ def _parse_data_model(self):
                     database = mnode["database"] if mnode["database"] else "default"
                     schema = mnode["schema"] if mnode["schema"] else "default"
                     raw_sql = mnode.get("raw_sql", "")
+                    description = mnode.get("description")
                     model = DataModel(
                         modelType=ModelType.DBT,
-                        description=mnode.get("description", ""),
+                        description=description if description else None,
```

Comment:
nit - this is a bit redundant, we could just use `decription` as it defaults to `None`

Reply:
Right but the use-case here is we are getting empty string in description which overwrites the `None` description set during metadata extraction and this results in generation of activity feed log, so here by `if description else None` I'm eliminating the empty string with `None`

Reply:
aaaah I see I see makes sense. Could we then please add a comment about it? Otherwise, we might "clean" it up at some point if we think this poorly as I am doing now :P